hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/proxy/SCMSecurityProtocolFailoverProxyProvider.java

```diff
@@ -99,7 +100,7 @@ public SCMSecurityProtocolFailoverProxyProvider(ConfigurationSource conf,
     this.retryInterval = scmClientConfig.getRetryInterval();
   }
 
-  protected void loadConfigs() {
+  protected synchronized void loadConfigs() {
```

Comment:
Do we need sychrnoize here?
As loadConfigs is called from constructor

Reply:
Same for all failOverProxy classes

Reply:
It was added to address findbug issues.